src/roiextractors/volumetricimagingextractor.py

```diff
@@ -0,0 +1,148 @@
+from typing import Tuple, List, Iterable, Optional
+import numpy as np
+
+from .extraction_tools import ArrayType, DtypeType
+from .imagingextractor import ImagingExtractor
+
+
+class VolumetricImagingExtractor(ImagingExtractor):
+    """Class to combine multiple ImagingExtractor objects by depth plane."""
+
+    extractor_name = "VolumetricImaging"
+    installed = True
+    installatiuon_mesage = ""
+
+    def __init__(self, imaging_extractors: List[ImagingExtractor]):
```

Comment:
Wiill all volumetric imaging extractors be a list of imaging extractors. We are dealing with a volumetric extractor that is not like that in the current conversion: two files for two channels and each has volumetric data.

Reply:
Yeah, so the idea would be to make a separate ImagingExtractor for each plane and then combine them with the VolumetricImagingExtractor.